### DISCUSSION TITLE
How to get the View port Extent with Avalonia MapControl

### DISCUSSION THREAD

**Discussion by trampster** [Mapsui/Mapsui#2509] [Q&A]
I need to determine if a feature is currently visible.

Other answers say to use  MapView.Map.Viewport.Extent.Contains

However I have an Avalonia.MapControl and there is no .Map.Viewport

**Comment by pauldendulk** [comment]
In v4 this is MapControl.Map.Navigator.Viewport.

It is mentioned in the upgrade guide.
https://mapsui.com/documentation/v4.0-upgrade-guide.html

Which other answers? I will include version info in those answers.

**Reply by trampster** [reply]
Sorry the other answers were places outside your control, stack overflow etc. Thanks for you help.
